src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/Header.js

```diff
@@ -1,12 +1,16 @@
 // @flow
 import React from 'react';
 import classNames from 'classnames';
+import Loader from '../Loader';
 import type {Button as ButtonConfig} from './types';
 import Button from './Button';
 import headerStyles from './header.scss';
 
+const LOADER_SIZE = 24;
```

Comment:
I guess you have clarified the size with the designer?

Reply:
No